iati/core/tests/test_schemas.py

```diff
@@ -2,56 +2,79 @@
 from lxml import etree
 import pytest
 import iati.core.codelists
+import iati.core.default
 import iati.core.exceptions
 import iati.core.schemas
 import iati.core.tests.utilities
 
 
-class TestSchemas(object):
-    """A container for tests relating to Schemas."""
+def default_activity_schema():
+    """Create a very basic acivity schema.
 
-    @pytest.fixture
-    def schema_initialised(self):
-        """Create a very basic Schema.
+    Returns:
+        iati.core.ActivitySchema: An ActivitySchema that has been initialised based on the default IATI Activity Schema.
 
-        Returns:
-            iati.core.Schema: A Schema that has been initialised with basic values.
+    """
+    schema_name = iati.core.tests.utilities.SCHEMA_ACTIVITY_NAME_VALID
 
-        """
-        schema_name = iati.core.tests.utilities.SCHEMA_NAME_VALID
+    return iati.core.default.schema(schema_name)
 
-        return iati.core.Schema(name=schema_name)
 
-    def test_schema_default_attributes(self):
-        """Check a Schema's default attributes are correct."""
-        schema = iati.core.Schema()
+def default_organisation_schema():
+    """Create a very basic organisaion schema.
 
-        assert schema.name is None
+    Returns:
+        iati.core.OrganisaionSchema: An OrganisaionSchema that has been initialised based on the default IATI Organisaion Schema.
 
-    @pytest.mark.parametrize("invalid_name", iati.core.tests.utilities.find_parameter_by_type(['str', 'none'], False))
-    def test_schema_name_instance(self, invalid_name):
-        """Check that an Error is raised when attempting to load a Schema that does not exist.
+    """
+    schema_name = iati.core.tests.utilities.SCHEMA_ORGANISATION_NAME_VALID
+
+    return iati.core.default.schema(schema_name)
 
-        Todo:
-            Check for type errors when the type is incorrect.
+
+class TestSchemas(object):
+    """A container for tests relating to Schemas."""
+
+    @pytest.fixture(params=[
+        iati.core.tests.utilities.SCHEMA_ACTIVITY_NAME_VALID,
+        iati.core.tests.utilities.SCHEMA_ORGANISATION_NAME_VALID
+    ])
+    def schemas_initialised(self, request):
```

Comment:
Only returns a single Schema on each invocation, so should be named (and documented to some extent) in a singular manner.